roles/openshift_openstack/templates/heat_stack_server.yaml.j2

```diff
@@ -228,7 +228,7 @@ resources:
       metadata:
         group: { get_param: group }
         environment: { get_param: cluster_env }
-        clusterid: { get_param: cluster_id }
+        clusterid: {{ openshift_openstack_clusterid }}
```

Comment:
Is this change needed? If we're passing in the wrong value, it's probably better to fix that.

Reply:
Ah, see above comment